lib/Doctrine/DBAL/Platforms/PostgreSqlPlatform.php

```diff
@@ -913,7 +915,7 @@ static function ($boolean) {
      */
     public function convertFromBoolean($item)
     {
-        if (in_array(strtolower($item), $this->booleanLiterals['false'], true)) {
+        if (in_array(strtolower((string) $item), $this->booleanLiterals['false'], true)) {
```

Comment:
Can it really be in upper case? Looks like it [can't](https://www.postgresql.org/docs/9.1/datatype-boolean.html). I'd better remove `(string)` and `strtolower()` altogether. If it's really needed, we'll have an integration test for that case.